graylog2-server/src/main/java/org/graylog2/periodical/IndexRetentionThread.java

```diff
@@ -145,6 +145,6 @@ public int getInitialDelaySeconds() {
 
     @Override
     public int getPeriodSeconds() {
-        return (int) MINUTES.toSeconds(5);
+        return (int) MINUTES.toSeconds(1);
```

Comment:
Did you change that on purpose? Or is it a leftover from testing?

Reply:
thanks for catching that